base/commands/migration/migration_start.go

```diff
@@ -95,5 +98,6 @@ Selected data structures in the source cluster will be migrated to the target cl
 }
 
 func init() {
+	cmd.SetCancelMsg(" (Ctrl+C to exit) ")
```

Comment:
It's better to put this at the start of `Exec`, so it doesn't affect other commands.

Reply:
https://github.com/hazelcast/hazelcast-commandline-client/pull/431/commits/58740c1075225cd14bf5664bd48909fb90f4ce80